src/app/core/data/object-updates/object-updates.service.ts

```diff
@@ -37,6 +43,10 @@ function filterByUrlAndUUIDFieldStateSelector(url: string, uuid: string): Memoiz
   return createSelector(filterByUrlObjectUpdatesStateSelector(url), (state: ObjectUpdatesEntry) => state.fieldStates[uuid]);
 }
 
+function virtualMetadataSourceSelector(url: string, source: string): MemoizedSelector<CoreState, VirtualMetadataSource> {
```

Comment:
Please add TypeDocs or a description for this function

Reply:
I didn't add these because the other selectors don't have typedocs neither.
If necessary I can still add them though.